src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
Haha - that's true. Lets play it safe and make the check a CollectionView method, that way for those who have two bb libs, they can override this one function to do the check appropriately and be happy.